source/widgets/ProjectStatus/js/configuration.js

```diff
@@ -29,13 +30,25 @@ function OctopusStatusWidgetConfiguration() {
                 return customSettings;
             };
 
-            var fetchDataSourceContent = function (queryUri, token, name, spaceId) {
-                let query;
+            var fetchDataSourceContent = function (queryUri, token, name, spaceId, partialNameFilter) {
+                let parameters = [];
```

Comment:
Modified this to support multiple parameters and have our query deal with multiple parameters.